nexus/src/external_api/console_api.rs

```diff
@@ -734,10 +766,6 @@ fn find_file(
         return Err(not_found("expected a non-directory"));
     }
 
-    if !file_ext_allowed(&current) {
-        return Err(not_found("file extension not allowed"));
-    }
-
```

Comment:
Moved this check out of `find_file` because it seemed silly to have to pop off `.gz` to do it. I also tried adding things like `js.gz` to the list of allowed extensions, but `extension()` only gives us the `.gz`. We _could_ make it work by looping through the list and using some kind `ends_with` thing.